### ISSUE TITLE
Settings area design

### ISSUE THREAD

**Issue by nclm** [tootsuite/mastodon#264]
* The navigation between initial profile state and the “Preferences” and “Change password” could be made clearer by turning them into tabs: Profile/Preferences/Account.
* The placeholders used as labels are an accessibility issue, and tends to question the use of the fields once you entered info inside. There should be clear labels.
* The link back to the main app isn’t evident, a smaller logo with a text label at the top left may be clearer.
* The background photo seems out of place, as it is just for decoration (this is a personal preference).
(The two last points can be applied to individual profiles and toots pages.)

![mastodon-3](https://cloud.githubusercontent.com/assets/923008/20608036/fa8642d8-b27d-11e6-9ff8-37321233771c.jpg)

**Comment by nclm** [comment]
A quick preview of how that could work.
![screen shot 2016-11-24 at 20 02 54](https://cloud.githubusercontent.com/assets/923008/20608326/1534de98-b281-11e6-95e1-827de6a6beba.png)

**Comment by Gargron** [comment]
I love the mockup, though I'm quite fond of the transparent-background and underline inputs; but that's a small difference.

How about the tabs at the top will take up the full width of the form, without spaces between lines (mirroring the design of posts/followers/follows on profiles), and the mastodon logo will be just the logo next to the tabs, same height as the tabs?

**Comment by nclm** [comment]
Yes, that could work!
The mock-up was simply a very quick playing with dev tools, just to get the idea, not the details :)

I think however that every piece of UI should be styled the same the same across the website – settings area, main app and individual pages included. So a tab should always be like other tabs, an input like other inputs, etc.

At one point, there’ll be a need to define the look of every module so they can all be cohesive together and be unified across all aspects of Mastodon. Without mentioning art direction and styleguides yet, the project is small enough to have this just being CSS, but something along these lines.

**Comment by triplefox** [comment]
A suggestion I just received is to use the fourth column panel to hold settings so that we aren't transitioning pages.

**Comment by Gargron** [comment]
Settings area has been redesigned!
